<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>250</y>
      <width>331</width>
      <height>131</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="btn_motor_f">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>正向</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_motor_b">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>逆向</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_motor_stop">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>关闭</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>331</width>
      <height>151</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>温度</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>湿度</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcd_temp">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcd_humi">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>50</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcd_lig">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>100</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>100</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>亮度</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>0</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>基于物联网试验环境的检测控制</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>310</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>控制：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>检测：</string>
    </property>
   </widget>
   <zorder>frame_2</zorder>
   <zorder>frame_4</zorder>
   <zorder>lineEdit</zorder>
   <zorder>label_10</zorder>
   <zorder>label_11</zorder>
   <zorder>btn_motor_b</zorder>
   <zorder>btn_motor_stop</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>676</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
